k8s/aat/software-engineering/data-extractor-job/data-extractor-job.yaml

```diff
@@ -13,7 +13,28 @@ spec:
     repository: https://hmcts.azurecr.io/helm/v1/repo/
     name: data-extractor-job
     version: 0.0.3
+  valueFileSecrets:
+  - name: "data-extractor-values"
   values:
     job:
       image: hmcts.azurecr.io/hmcts/data-extractor-job:prod-f888e665
       suspend: true
+      keyVaults:
+        "rpe-prod":
+          resourceGroup: rpe-prod
+          secrets:
+            - aat-ccdro-user
```

Comment:
this shouldn't be retrieved from rpe-prod, I only suggested it as somewhere for it to put so we can receive it.

A temp MI vault should be created, once the MI subscription is setup for its AKS cluster the secret can be moved there